tools/getkeys/package.json

```diff
@@ -24,7 +24,7 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../.prettierignore",
 		"start": "node ./index.js",
 		"test": "echo \"Error: no test specified\" && exit 1",
-		"tsc": "tsc"
+		"tsc": "echo \"No build required.\""
```

Comment:
Can we not just remove this script?

Reply:
I had no confidence in that. I copied this from changelog-generator-wrapper and expected there was a reason to have it. Will try removal.